bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -362,12 +362,14 @@ public synchronized void sttEventReceived(STTEvent sttEvent) {
             logger.debug("RecognitionStopEvent event received");
             toggleProcessing(false);
         } else if (sttEvent instanceof SpeechRecognitionErrorEvent sre) {
-            logger.debug("SpeechRecognitionErrorEvent event received");
+            logger.warn("SpeechRecognitionErrorEvent event received: {}", sre.getMessage());
             if (!isSTTServerAborting) {
                 abortSTT();
                 toggleProcessing(false);
-                String text = i18nProvider.getText(bundle, "error.stt-error", null, dialogContext.locale());
-                say(text == null ? sre.getMessage() : text.replace("{0}", sre.getMessage()));
+                if (sre.getNotify()) {
+                    String text = i18nProvider.getText(bundle, "error.stt-error", null, dialogContext.locale());
+                    say(text == null ? sre.getMessage() : text.replace("{0}", sre.getMessage()));
```

Comment:
Maybe your problem was the injection of the message coming from `SpeechRecognitionErrorEvent`  inside a more global message defined by the entry "error.stt-error" ?
Maybe you should say simply the message coming from `SpeechRecognitionErrorEvent` ?